content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/review-and-streamline-feedback-and-marking.html.erb

```diff
@@ -0,0 +1,136 @@
+            <div class="govuk-grid-column-full">
+              <div class="info-box govuk-!-margin-top-0 govuk-!-margin-bottom-0">
+                <div class="info-box__corner">
+                  <img src="/assets/images/download-icon.svg" alt="Bullseye icon">
+                </div>
+                <h2 class="govuk-heading-m">
+                  Download the presentation
+                </h2>
+                <div class="govuk-grid-row" style="align-items: center;">
+                  <div class="govuk-grid-column-one-half">
+                    <img src="/assets/images/preview-placeholder.jpg" alt="Placeholder image"
+                      class="dfe-file-preview-image">
+                  </div>
+                  <div class="govuk-grid-column-one-half">
+                    <div class="info-box__content">
+                      <p class="govuk-body-s govuk-!-margin-bottom-6">
+                        <strong>Presentation length:</strong> 1 hour and 15 minutes
+                      </p>
+                      <p class="govuk-body-s govuk-!-margin-bottom-6">
+                        <strong>Resource type:</strong> Presentation
+                      </p>
+                      <p class="govuk-body-s govuk-!-margin-bottom-6">
+                        <strong>File type:</strong> OpenDocument Presentation, 171KB, 7 pages
+                      </p>
+                      <a class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold"
+                        href="<%= @base_url %>/assets/files/Review and streamline feedback and marking.odp">
+                        Download
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </ul>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          Audience
+        </h2>
+
+        <p>
+          This resource has been provided to support senior leadership teams to run a session with the whole school, teams or departments.   
+        </p>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          Presentation length
+        </h2>
+
+        <p>
+          The presentation should take 1 hour and 15 minutes. You should also allocate time for the facilitator to prepare the
+          session for use in your school.
+        </p>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-7">
+          How to run with your staff
+        </h2>
+
+        <p>
+          Depending on the situation in your school, you might wish to use this workshop as a starting point to initiate further exploration, or to prepare for the workshop with activities beforehand.  
```

Comment:
This sentence should be broken down into 2: 
'Depending on the situation in your school, you might wish to use this workshop as a starting point to initiate further exploration. You can also use it to prepare for the workshop with activities beforehand.'